Makefile

```diff
@@ -655,9 +655,9 @@ external_eigen: $(EIGEN_BASE)
 
 $(EIGEN_BASE):
 	# from Eigen master branch on gitlab as of 2022.08.09
-	git clone https://gitlab.com/AuroraPerego/eigen.git $@
-	# include all SYCL updates
-	cd $@ && git reset --hard eb029b6bc9c33172f0b77524d226d2bca52945a2
+	git clone https://github.com/makortel/eigen.git $@
```

Comment:
@fwyzard I wonder if we should setup eigen repository under cms-patatrack organization

Reply:
Sure, we can do that.
Eventually we should push the changes to the CMSSW version...

Reply:
(... or upstream ...)